Gordon360/Authorization/StateYourBusiness.cs

```diff
@@ -188,6 +188,11 @@ private async Task<bool> CanReadOneAsync(string resource)
                     }
                     return false;
                 }
+            case Resource.HOUSING_CONTACT_PREFERENCE:
+                return (user_groups.Contains(AuthGroup.RD) || user_groups.Contains(AuthGroup.HousingAdmin) ||
+                    user_groups.Contains(AuthGroup.HallInfoViewer) || user_groups.Contains(AuthGroup.Student));
```

Comment:
Students need to be able to see Contact preference for the on call RA for their hall which is not necessarily "Their" RA. When it comes to the village, students technically need to see ra info from another hall as the RA on call covers multiple dorms, this makes securing this data a little difficult on a per dorm basis

Reply:
Thinking about this a little further, it would make sense that any student should be able to contact the on-call RA for any hall, in the sense that a student who doesn't live in Nyland but is currently in Nyland in a situation that requires the RA on-call should be able to contact said RA.